pkg/lvm/lvm_util.go

```diff
@@ -59,6 +60,34 @@ const (
 	YES = "yes"
 )
 
+// LogicalVolume specifies attributes of a given lv that exists on the node.
+type LogicalVolume struct {
```

Comment:
The `json` in this struct is not required as we are not doing a json parsing directly here.